components/script/dom/bluetoothuuid.rs

```diff
@@ -12,14 +12,14 @@ use crate::dom::bindings::reflector::Reflector;
 use crate::dom::bindings::str::DOMString;
 use crate::dom::window::Window;
 
-#[allow(clippy::upper_case_acronyms)]
```

Comment:
Doesn't removing this allow for create a new warning for the `UUID` type?

Reply:
it does indeed! sorry for the overlook, I'll fix it.